README.md

```diff
@@ -4,183 +4,7 @@ Looking to do some Rust development on Tessel? You’ve found the right place.
 
 **Get in touch**: the team working on Rust for Tessel is all on the #rust-lang channel of the [Tessel Slack ![](https://tessel-slack.herokuapp.com/badge.svg)](https://tessel-slack.herokuapp.com/). Join the conversation!
 
-**Best place to start right now:** [This snippet](https://gist.github.com/johnnyman727/f76616edc71a8646b27b27809ea464c6), published July 5th (2016) is the most helpful introduction at the moment.
+**Best overview for getting started right now:** [This snippet](https://gist.github.com/johnnyman727/f76616edc71a8646b27b27809ea464c6), published July 5th (2016) is the most helpful introduction at the moment.
```

Comment:
Better to move it out of the snippet & into the readme so that others can edit when they need to.

Also I think it's been edited since July 5, correct?